app/src/main/java/com/duckduckgo/app/pixels/EnqueuedPixelWorker.kt

```diff
@@ -63,12 +66,17 @@ class EnqueuedPixelWorker @Inject constructor(
             return
         }
         Timber.i("Sending app launch pixel")
+        val collectWebViewFullVersion = browserTelemetryConfig.shouldCollectOnAppLaunch()
+        val paramsMap = mutableMapOf<String, String>().apply {
```

Comment:
I think `BrowserTelemetryConfig` should directly return a map with the parameters that we should add and checking if we should be collecting on launch should be an implementation detail.